File: semester 4/MPP/JFTS/FestivalPersistance/src/main/java/ro/mpp/repository/IShowArtistRepository.java
package ro.mpp.repository;

import ro.mpp.domain.ShowArtist;

import java.time.LocalDate;
import java.util.List;

public interface IShowArtistRepository extends Repository<Integer, ShowArtist> {
    public List<ShowArtist> filterByDate(LocalDate date);
}

File: semester 4/MPP/C#FTS/Model/Repository/IShowArtistRepository.cs
using C_FTS.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace C_FTS.Repository
{
    public interface IShowArtistRepository : IRepository<int, ShowArtist>
    {
        public List<ShowArtist> FilterByDate(DateTime date);
    }
}
